### ISSUE TITLE
backspace should set the setPauseStatus to typewrite=false

### ISSUE THREAD

**Issue by jsgoupil** [mattboldt/typed.js#472]
### Description

I'm trying to start/stop a couple of Typed together. I understand there are a lot involved with the timers, however, I believe there is a bug on the line where the `backspace` starts but it is paused state.

```
this.setPauseStatus(curString, curStrPos, true);
```

The 3rd parameter should be `false` in the `backspace` function.

### Demo
Obviously this demo could be made better with a `backDelay`. But I want to show the concept of start/stop not handling it properly. This code will never show the second sentence.

https://jsfiddle.net/wq2v047j/2/
```
var typed = new Typed('#typed', {
  strings: ['First sentence.', 'Second sentence.'],
  onStringTyped: (arg1, arg2) => {
    console.log('Called');
    typed.stop();

    setTimeout(() => typed.start(), 1000);
  }
});
```

### Steps to Reproduce

1. Run the script above.

**Expected behavior:**
The backspace sequence should start after 1000ms after the first sentence has been written.

**Actual behavior:**
Nothing visually happens. But the `onStringTyped` is called constantly.

**Reproduces how often:**
100%

### Additional Information
My ugly work around is to reset your internal state before I restart.
```
setTimeout(() => {
    arg2.pause.typewrite = false;
    typed.start()
}, 1000);
```

**Comment by mattboldt** [comment]
Thank you for the detailed report! Yup, it looks like this is a bug:
https://github.com/mattboldt/typed.js/blob/646952b3e6e24adb1dd63276fcf465eb9caedf21/src/typed.js#L243-L247
The 3rd arg should definitely be `false`. Working on a fix now!

**Comment by mattboldt** [comment]
Fixed and published in v2.0.12!
